benches/benches/transaction_throughput.rs

```diff
@@ -179,16 +181,22 @@ fn signed_transfers(c: &mut Criterion) {
                 Default::default(),
                 Default::default(),
             )
-            .add_unsigned_coin_input(
-                SecretKey::random(rng),
+            .add_input(Input::coin_predicate(
                 rng.gen(),
+                owner,
                 1000,
                 Default::default(),
                 Default::default(),
-            )
+                Default::default(),
+                predicate.clone(),
+                vec![],
+            ))
             .add_output(Output::coin(rng.gen(), 50, AssetId::default()))
             .add_output(Output::change(rng.gen(), 0, AssetId::default()))
-            .finalize()
+            .finalize();
+        tx.estimate_predicates(&checked_parameters(), MemoryInstance::new())
+            .expect("Predicate check failed");
```

Comment:
What's the purpose of estimating the predicates here?

Reply:
We don't use predicate instead. We use one signed coin along with one predicate. All benchmarks now use at least one predicate that returns `true`. The second input is different and the name reflects it